src/VisualStudio/Core/Def/Implementation/AnalyzerDependencyCheckingService.cs

```diff
@@ -46,7 +46,7 @@ internal sealed class AnalyzerDependencyCheckingService
             {
                 results = await GetConflictsAsync().ConfigureAwait(continueOnCapturedContext: true);
             }
-            catch (OperationCanceledException)
+            catch
```

Comment:
@sharwell Unfortunately `ErrorHandler.IsCriticalException` includes `BadImageFormatException`, which is a (relatively) likely exception to come out of `GetConflictsAsync`--e.g., if someone manages to pass in something that looks like a managed DLL but isn't.

Still, the general point stands, and I'll have to think about the right thing to do here. In general we don't have a consistent story around capturing and reporting exceptions in the IDE.